virttest/qemu_devices/qcontainer.py

```diff
@@ -580,6 +580,16 @@ def clean(device, added_devices):
         added_devices.append(device)
         return added_devices
 
+    def is_pci_device(self, device):
+        """
+        check if the device is plugged into pci bus
+        :param device: device name
+        :return: True if is pci device, otherwise False
+        """
+        return bool(re.search(
+            r'name "%s", bus PCI|bus PCI, .*alias "%s"' % (device, device),
```

Comment:
Not all devices have "alias ...", eg:
name "e1000", bus PCI, desc "Intel Gigabit Ethernet"
name "rtl8139", bus PCI

Reply:
@huangyum Yes, so there is an or pattern: `|`, so it actually works for the same no alias pattern `name "pc-dimm", bus PCI`

Reply:
I see.  Why are the name and alias using same 'device', typo?

Reply:
@huangyum No, that's intended, because we might use the real name or the alias name as the device name, e.g. `"virtio-blk-pci"` alias `"virtio-blk"`, so no matter avocado use which one of them, this pattern will accept it, say, `device =="virtio-blk-pci"` or `device =="virtio-blk"`

Reply:
@huangyum Similar with https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qcontainer.py#L400

Reply:
Thanks for your explanation.  I just misunderstood '|'.  You are using it correctly.